### DISCUSSION TITLE
Comment link lost

### DISCUSSION THREAD

**Discussion by MorganReid** [outline/outline#5985] [Self Hosting] [answered]
User A wrote a `comment` on a certain content of the document(No edit mode). At this time, the content is marked with a blue underline. Clicking on this content will expand the `comments` on the right side. Of course, clicking on the `comment` can also jump to the corresponding content of the document.
![image](https://github.com/outline/outline/assets/52235796/9101bc85-aa46-46dc-b49c-fe51340887c1)

But at this time, user B opened the same document and could not find the content marked with a blue underline. If i expand the `comments` on the right, i can see the `comment`, but clicking the `comment` at this time cannot jump to the content corresponding to the document.
![2023 10 11_16 55 34](https://github.com/outline/outline/assets/52235796/cb9a1144-f7b7-42b3-aebc-27cd6df2bb56)

Through the console, I found that the `comment-marker `will be added to the commented content. This does not exist when it cannot be jumped. But I don’t know what exactly this does?
![image](https://github.com/outline/outline/assets/52235796/f9a37cf7-22b6-48f8-91f7-bdfedade523b)

I queried the database and found the `documents` and `comments` tables, but it seems that the mapping relationship between comments and corresponding content is not recorded in it. What is the cause of this situation? How should I troubleshoot?What is the principle of outline to jump from comment to content?

**Comment by NangeGao** [comment]
The same for me.

**Comment by tommoor** [comment, marked as answer]
This was fixed in https://github.com/outline/outline/issues/6011 – will be in the next patch release soon.
